src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -33,7 +45,99 @@ const BlastSubmissionResults = () => {
 };
 
 const Main = () => {
-  return <div>Here will be submission details</div>;
+  const location = useLocation();
+  const submissionId = location.pathname.split('/').slice(-1)[0];
+  const blastSubmission = useAppSelector((state) =>
+    getBlastSubmissionById(state, submissionId)
+  );
+
+  return (
+    <div className={styles.blastSubmissionResultsContainer}>
+      <Header submission={blastSubmission} isAnyJobRunning={false} />
+      <SequenceBox submission={blastSubmission} />
+    </div>
+  );
+};
+
+type SequenceBoxProps = {
+  submission: BlastSubmission;
+};
+
+const SequenceBox = (props: SequenceBoxProps) => {
```

Comment:
Why does a component called `SequenceBox` return an array of sequence boxes?

Reply:
I would like to call it a submission box because it is a